gensim/__main__.py

```diff
@@ -1,50 +1,57 @@
+"""Copyright (C) 2016 ExplosionAI UG (haftungsbeschränkt), 2016 spaCy GmbH,
+2015 Matthew Honnibal
+https://github.com/explosion/spaCy/blob/master/LICENSE
```

Comment:
List the explicit license, not a link (a link can change in time, go dead etc).

Also, if you're making changes, be sure to include yourself in the authors and copyright (say "adapted from ..., original author ..., license ...").